runelite-api/src/main/java/net/runelite/api/widgets/WidgetInfo.java

```diff
@@ -153,6 +153,8 @@
 	BANK_SETTINGS_BUTTON(WidgetID.BANK_GROUP_ID, WidgetID.Bank.SETTINGS_BUTTON),
 	BANK_TUTORIAL_BUTTON(WidgetID.BANK_GROUP_ID, WidgetID.Bank.TUTORIAL_BUTTON),
 
+	GROUP_STORAGE_CONTAINER(WidgetID.GROUP_STORAGE_GROUP_ID, WidgetID.GroupStorage.CONTAINER),
+	GROUP_STORAGE_TITLE_BAR(WidgetID.GROUP_STORAGE_GROUP_ID, WidgetID.GroupStorage.TITLE),
```

Comment:
TITLE_BAR isn't exactly right here. This widget seems to contain the UI elements of the window--including the title bar, but also the window background, edges and corners, and title line. Perhaps `GROUP_STORAGE_UI_CONTAINER` is a better name?